test/javascript/components/mentoring/mentor_conversations_list/pagination.test.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import '@testing-library/jest-dom/extend-expect'
+import Pagination from '../../../../../app/javascript/components/mentoring/mentor_conversations_list/pagination.jsx'
+
+test('clicking on "First" sets page to 1', () => {
+  const setPage = jest.fn()
+  const { getByText } = render(
+    <Pagination current={2} total={10} setPage={setPage} />
+  )
+
+  userEvent.click(getByText('First'))
```

Comment:
Usually this uses `fireEvent` so you don't need `userEvent`.

Reply:
Cool. Thanks! I've updated the tests now.